tests/Fixtures/TestBundle/Serializer/Denormalizer/DiscrAbstractDummyDenormalizer.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
The problem resides in the fact that the collection is a collection of DiscrAbstractDummy (which is, abstract) and thus cannot be instancied directly. This require a similar functionality as in the JMS serializer (or doctrine) to be implemented in the SF serializer to be able to denormalize correctly out of the box